source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -212,6 +215,7 @@ enum class CardElementType
     AdaptiveCard,
     ChoiceInput,
     ChoiceSetInput,
+    ChoicesData,
```

Comment:
ChoicesData doesn't have UI to render, so it's better not to include here and the enum value is not used anywhere.

Reply:
removed it.